settings.gradle

```diff
@@ -1,2 +1,3 @@
+include ':shoplist'
```

Comment:
By the way, why do you need this module?
Most of the code you are doing in app module.
So it is better to delete shoplist module or remove most of the logic from app to shoplist module